projects/ama-sdk/src/lib/store/project-data.state.ts

```diff
@@ -15,15 +15,32 @@
  * limitations under the License.
```

Comment:
I don't think this state belongs to the `ProjectDataState` which is part of the `ProjectEditorState` (when we are already inside one project). I think this should be inside the DashboardState, somehow like this:

```
export interface DashboardState {
    projects: ProjectSummaryEntities;
    loading: boolean;
    projectsLoaded: boolean;
    pagination: Pagination;
    releases: ReleasesSummaryEntities;
    loadedReleases: boolean;
    releasesPagination: Pagination;
    ...
}
```